addon/components/fl-input.ts

```diff
@@ -50,6 +52,11 @@ export default class FlInput<T extends FlInputArgs> extends Component<T> {
         return this.args.id ?? `fl-input-${guidFor(this)}`;
     }
 
+    get inputName() {
+        const name = this.args.name ?? this.args.placeholder;
+        return name ? dasherize(name) : '';
```

Comment:
prob shouldnt touch a manually passed in `name` arg. e.g.
```ts
get inputName() {
    return this.args.name ?? dasherize(this.args.placeholder);
}
```

also i forget, but does `dasherize()` handle stripping out non alphanumeric symbols for you too? e.g. for like `"Bob's Payment Amount ($)" => "bobs-payment-amount"`

Reply:
Turns out it doesn't. Updated that to remove non-word characters as ^ ended up as "bob's-payment-amount-($)"